makefile

```diff
@@ -432,7 +432,7 @@ cppcheck:
 sonar_metrics: coverage
 	scripts/build/sonarProperties.sh $(BROKER_VERSION) > sonar-project.properties 
 	cd BUILD_COVERAGE/src && gcovr --gcov-exclude='.*parseArgs.*' --gcov-exclude='.*logMsg.*' -x -o ../../coverage.xml && cd ../../
-	sed s#filename=\"/var/develenv/jenkins/jobs/orion-sonar/workspace/#filename=\"/var/develenv/jenkins/jobs/metrics-queue-consumer/workspace/workspace/#g ./coverage.xml > ./coverage_sonar.xml
+	sed 's#filename="/var/develenv/jenkins/jobs/'$(JOB_NAME)'/workspace#filename="/var/develenv/jenkins/jobs/metrics-queue-consumer/workspace/workspace#g' coverage.xml > coverage_sonar.xml
```

Comment:
I'd suggest to add a comment about this line explaining that the JOB_NAME env variable is typically injected by jenkins when executing this job.